backend/app/api_views.py

```diff
@@ -80,8 +80,42 @@ def numbers_to_music(request):
 # COLOR
 ################################################################################
 @api_view(['POST'])
-def color_to_music(_request):
-    return not_implemented_error()
+def color_to_music(request):
+    """
+    :param request: color picker's rgb values stored as a dictionary when the user hits submit
+    :return: wav file, sine wave with frequency corresponds to energy of the colors
+    """
+    response_object = request.data['listOfColors']
+
+    sound = None
+    for response in response_object:
+        r = response["r"]
```

Comment:
This is duplicated entirely from `color_to_samples` -- let's factor out the common code.